### DISCUSSION TITLE
lolMiner 1.68

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1869] [General]
### Changes
- Significantly improved Nexa mining performance on supported GPUs (e.g. up to  +25% on 8G Ampere GPUs)

### Fixes
- Fixed a bug that can make the miner crash with error message "Authorization problem on all configured pools 1" during Nexa mining

General note: The Nexa gpu codes are (and will be in foreseeable future) joint work with Iedoc from [BzMiner](https://github.com/bzminer/bzminer). Miner fees will be evenly shared regardless of which of these two flavors of the code you prefer.  

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.68'>lolMiner 1.68</a>.</em>

**Comment by tanerileri69** [comment]
Have amd card holders done you a disservice?
why are you constantly making improvements for nvidia?
We request that amd support be added to nexa as soon as possible.

**Reply by Lolliedieb** [reply]
Deleted this comment 2 times, for some reason it was here multiple times.
The amd code is in development and it was planned for 1.68, but the appearance of an other miner pushing its Nvidia release gave us the feeling we need to react in order not to loose too much market share. Still amd solver is work in progress and the progress is not that bad, so I am sure we will have it with the next release.

**Reply by tanerileri69** [reply]
Thanks for your kind reply.
We are eagerly waiting for AMD support.
I trust lolminer, I believe you will do the best and not lose your market share.

**Comment by saki8b** [comment]
It's getting removed by Windows defender 
is it safe to allow it?

**Comment by rickfan1** [comment]
"error in allocating memory 0" for my 3070ti
but 3080 is normal.
please help me

**Comment by AnthonyAmaru** [comment]
Hey my 3070tis are actining weird now. They were all fine before mining at the same hashrate now they are dropping in hashrate. Also, I had all my GPUs running at a great efficieny before and now the effeciency has droppped by a good amount. I like more hashrate but not for more electric use if the effecieny is not there.

**Comment by romi0857** [comment]
"error in allocating memory 0" for my 3070
but 3090 is normal.my win10 64
please help me

**Comment by kimghwa** [comment]
My rtx3070ti is strange. In lolminer 1.67 version, 57 hashes came out, but in lolminer 1.68 version, only 50 hashes. Please check.

**Comment by tam-wh** [comment]
Upgraded from 1.67 to 1.68, hashrate drops by a lot
3080Ti - 102 > 78
3080 - 80 > 65

**Reply by artdekortim** [reply]
12Х 3060ti 
error in allocating
![image](https://user-images.githubusercontent.com/125633189/221557745-dae3471b-11df-4573-9e0c-dc5705b8a487.png)

**Reply by artdekortim** [reply]
lolMiner_v1.68_alternative_Win64   
12Х 3060ti  work well !)

**Comment by gavinmcroy** [comment]
I'm getting strange behavior, my 3080ti's when mining Nexa. Dropped from 90 to around 77 and all of them are locked there. Issue only on my 8 card rig, on my PC with exact same card model on same OC settings I'm getting 116 mh/sec
